clientlibs/js/src/functions/getExperimentCondition.ts

```diff
@@ -5,16 +5,16 @@ export class Assignment {
   private _payloadType: PAYLOAD_TYPE;
```

Comment:
You have missed updating the return type of `getPayload` and `getFactorPayload` to `IPayload | null`